manifests/09_metrics_service.yaml

```diff
@@ -0,0 +1,23 @@
+# Expose operator metrics
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    service.alpha.openshift.io/serving-cert-secret-name: serving-cert
+  labels:
+    app: cluster-storage-operator
+  name: metrics
+  namespace: openshift-cluster-storage-operator
+spec:
+  ports:
+  - name: https
+    port: 443
+    protocol: TCP
+    targetPort: 8443
+  selector:
```

Comment:
I think CSO's metric setup is busted. In deployment we say: 

```
          ports:
          - containerPort: 60000
            name: metrics
```

And then since we don't specify port in startup command, it will default to 8443, but then where does 443 comes from?

Reply:
* `8443` is where library-go listens: https://github.com/openshift/library-go/blob/9821002633e882489381e51d1bb4442788c0c01f/pkg/config/configdefaults/config_default.go#L46.
* `443` is where we want to expose the metrics service.

Good point about the `containerPort`, fixed to 8443.

Reply:
we as such don't have to change the port I think. We can as well leave it same as target port I think.